kobo-uncaged/static/ku.js

```diff
@@ -360,6 +372,16 @@ function handleShowKUCfg(resp) {
         document.getElementById('preferSDCard').checked = kuConfig.opts.preferSDCard;
         document.getElementById('preferKepub').checked = kuConfig.opts.preferKepub;
         document.getElementById('enableDebug').checked = kuConfig.opts.enableDebug;
+        //document.getElementById('excludeFormats').value = kuConfig.opts.excludeFormats.toString();
+        var formatLabels = document.querySelectorAll('#excludeFormatsContainer label');
+        for(var i = 0; i < formatLabels.length; i++) {
+            var lbl = formatLabels[i];
+            var chk = document.getElementById(lbl.htmlFor);
+            chk.checked = true;
+            if (kuConfig.opts.excludeFormats.indexOf(lbl.innerText) >= 0) {
+                chk.checked = false;
```

Comment:
I'd combine these lines instead of having an if statement just to set a boolean.

Reply:
I tend to be a bit verbose the first time I code something.

Hopefully what I've just committed isn't too terse now.